sig/datadog/appsec/waf.rbs

```diff
@@ -93,15 +93,8 @@ module Datadog
           end
         end
 
-        class RuleSetInfo < ::FFI::Struct
-        end
-
-        RuleSetInfoNone: ::Datadog::AppSec::WAF::LibDDWAF::RuleSetInfo
-
-        def self.ddwaf_ruleset_info_free: (RuleSetInfo) -> void
-
-        def self.ddwaf_init: (top, Config, RuleSetInfo) -> ::FFI::Pointer
-        def self.ddwaf_update: (::FFI::Pointer, LibDDWAF::Object, RuleSetInfo) -> ::FFI::Pointer
+        def self.ddwaf_init: (top, Config, Object) -> ::FFI::Pointer
+        def self.ddwaf_update: (::FFI::Pointer, LibDDWAF::Object, Object) -> ::FFI::Pointer
```

Comment:
Minor: It's kinda weird to be mixing `LibDDWAF::Object` and `Object`, as it makes them look like different things ;)